src/box/sql/sqlInt.h

```diff
@@ -2064,6 +2064,8 @@ struct Parse {
 		struct drop_view_def drop_view_def;
 		struct enable_entity_def enable_entity_def;
 	};
+	/** Number of the new BOX constraints in CREATE TABLE or ADD COLUMN. */
+	uint32_t constr_count;
```

Comment:
Can you reuse `autoname_i`?

Reply:
For now decided to leave it the way it was before.